kong/api/routes/cluster.lua

```diff
@@ -1,57 +1,125 @@
 local singletons = require "kong.singletons"
 local responses = require "kong.tools.responses"
+local cache = require "kong.tools.database_cache"
+local event_types = require("kong.core.events").TYPES
 
 local pairs = pairs
 local table_insert = table.insert
 local string_upper = string.upper
 
+local OTHER = "other"
+
+local function get_members(filters)
+  local members, err = singletons.serf:members()
+  if err then
+    return nil, err
+  end
+
+  local result = {}
+  for _, member in pairs(members) do
+    local matches = true
+    if filters then
+      for k, v in pairs(filters) do
+        matches = matches and member[k] == v
+      end
+    end
+    if matches then
+      table_insert(result, {
+        name = member.name,
+        address = member.addr,
+        status = member.status
+      })
+    end
+  end
+
+  return result
+end
+
+local function increment(key, value)
+  local ok, err = cache.incr(key, value)
+  if not ok and err == "not found" then
+    cache.rawset(key, value)
+  elseif not ok then
+    ngx.log(ngx.ERR, err)
+  end
+end
+
 return {
   ["/cluster/"] = {
     GET = function(self, dao_factory, helpers)
-      local members, err = singletons.serf:members()
-      if err then
-        return responses.send_HTTP_INTERNAL_SERVER_ERROR(err)
+
+      local events_stat = {
+        total = cache.get(cache.event_key()),
+        other = cache.get(cache.event_key(OTHER))
```

Comment:
`other` is really lacking context. A user reading that response would have no idea what it is, we should give a hint or name it slightly better. 
It sounds really weird to have a `total` field, and an `other` field. Shouldn't `other` be included in `total`? This really is lacking some more context.